src/platforms/hosted/platform.h

```diff
@@ -67,7 +67,16 @@ void platform_buffer_flush(void);
 #define PRODUCT_ID_STLINKV3        0x374fU
 #define PRODUCT_ID_STLINKV3E       0x374eU
 
-#define VENDOR_ID_SEGGER 0x1366U
+#define VENDOR_ID_SEGGER   0x1366
+#define PRODUCT_ID_UNKNOWN 0xFFFF
```

Comment:
Hex constants should be lower case (0xffff)

Please suffix all of these with `U` to make them unsigned (`-Wconversions` lint that will become enabled in the future)